File: graph.chat.server.entities/Message.cs
namespace graph.chat.server.entities;

public class Message
{
    public Guid Id { get; set; }
    public Topic? Topic { get; set; }
    public Guid TopicId { get; set; }
    public Guid SenderId { get; set; }
    public string? Content { get; set; }
}

File: src/graph.client/src/models/message.ts
export class Message{
    public Id: string;
    public TopicId: string;
    public SenderId: string;
    public Content: string;
    public Timestamp: string;
    
    constructor(id: string, topicId: string, senderId: string, content: string, timestamp: string) {
        this.Id = id;
        this.TopicId = topicId;
        this.SenderId = senderId;
        this.Content = content;
        this.Timestamp = timestamp
    }
}